libs/features/jarvis-chat.js

```diff
@@ -278,23 +281,31 @@ const startInitialization = async (config, event) => {
   });
 };
 
-const initJarvisChat = async (config, loadScriptFunction, loadStyleFunction) => {
+const initJarvisChat = async (
+  config,
+  loadScriptFunction,
+  loadStyleFunction,
+  getMetadataFunction,
+) => {
   if (!config?.jarvis) return;
 
   loadScript = loadScriptFunction;
   loadStyle = loadStyleFunction;
+  getMetadata = getMetadataFunction;
+
+  const onDemandMeta = getMetadata('jarvis-on-demand')?.toLowerCase();
+  const onDemand = onDemandMeta ? onDemandMeta === 'on' : config.jarvis.onDemand;
```

Comment:
I think this could just be

```javascript
const onDemand = onDemandMeta === 'on' || config.jarvis.onDemand
```

Reply:
if onDemandMeta is 'off' and config.jarvis.onDemand is true. Then `onDemand` should be false.